HotelReservation/Business/DTO/LoginUserDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Business.DTO
```

Comment:
DTO is a pattern (abstract name) -> rename to smth meaningfull

Reply:
renamed